feat: add few faqs (#2400)
novuhq/novu: docs/docs/platform/subscribers.md

## Subscriber Preferences

Novu manages a data model to help your users configure their preferences in an easy way. You can learn more about this in the [Subscriber Preferences](/platform/preferences) section.

## Frequently Asked Questions

<details>
  <summary>How to store custom properties in subscriber</summary>
  <p>Subscribers have fixed schema. Storing custom properties is not supported but here is a work around, you can store that property in your database and send those values in payload option of trigger using variables. Read more about variables <a href="./templates#variable-usage"> here </a>.</p>
</details>

<details>
  <summary>How to get subscriber properties before step execution in workflow</summary>

>   <p>Workflow has access to all existing properties of subscriber as well as payload variables. So there is no any extra step needed</p>

Reviewer: ```suggestion
  <p>Workflow has access to all existing properties of subscriber as well as payload variables. So no extra steps are needed</p>
```
